src/pages/Home.test.tsx

```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import { render, screen, waitFor } from '@testing-library/react';
 import Home from './Home';
 import { CheckSummaries, Severity } from 'types';
+import { MemoryRouter } from 'react-router-dom';
+
+// Helper function to render with router
+export const renderWithRouter = (ui: React.ReactElement, { route = '/' } = {}) => {
+  return render(
+    <MemoryRouter future={{ v7_relativeSplatPath: true, v7_startTransition: true }} initialEntries={[route]}>
+      {ui}
+    </MemoryRouter>
+  );
+};
```

Comment:
nit: I'd consider moving this to a separate file to avoid jest importing everything in here in other tests which might create weird circular deps or mocks in the other tests.